source/mainnet/net/desktop-wallet/install-ledger-app.rst

```diff
@@ -2,40 +2,40 @@
 .. _install-Ledger-app:
 
 ==============================================================
-Set up the Ledger Nano S and install the Concordium Ledger App
+Set up the Ledger device and install the Concordium Ledger App
 ==============================================================
 
-To be able to sign and send transactions using the Desktop Wallet, you need a Ledger Nano S hardware wallet, and you need to install the Concordium Ledger Application on the Ledger. You can use Windows, macOS or Ubuntu to install the app.
+To be able to sign and send transactions using the Desktop Wallet, you need a Ledger hardware wallet, and you need to install the Concordium Ledger Application on the Ledger. You can use Windows, macOS or Ubuntu to install the app.
 
 .. Note::
 
-   The Ledger Nano S Plus and Ledger Nano X are not supported currently.
+   The Ledger Nano X is not supported currently.
 
 Prerequisites
 =============
 
 -  Ledger hardware wallet
 
-.. Warning:: During the process described in this guide, you’ll generate private keys on the Ledger Nano S hardware wallet, and you’ll receive a 24-word recovery phrase. This is the only backup of your private keys. Make sure that you store it securely.
+.. Warning:: During the process described in this guide, you’ll generate private keys on the Ledger hardware wallet, and you’ll receive a 24-word recovery phrase. This is the only backup of your private keys. Make sure that you store it securely.
 
-Set up the Ledger Nano S
+Set up the Ledger device
 ========================
 
-The Ledger Nano S will generate the unique 24-word recovery phrase that is used to derive your private keys.
+The Ledger device will generate the unique 24-word recovery phrase that is used to derive your private keys.
 
 #. Download and install **Ledger Live**. For information on how to do this, see `Ledger's documentation <https://www.ledger.com/ledger-live/download>`_. You’ll only need Ledger Live when you set up the Ledger and update the firmware.
 
-#. Open **Ledger Live**, select **Get started**, and then select **Nano S**.
+#. Open **Ledger Live**, select **Get started**, and then select **Nano S** or **Nano S Plus**.
 
 #. Follow the on-screen setup instructions to set up your PIN code on the Ledger.
 
 #. Follow the on-screen instructions to get your 24-word recovery phrase.
 
 .. Warning:: Make sure that you write down the recovery phrase precisely as displayed and in the correct order. The recovery phrase is the only backup of your private keys.
 
-Once you've set up the Ledger, you must check that it's running the proper firmware version. The Concordium Ledger App currently supports Ledger firmware version 2.0.0 and 2.1.0 on **Mainnet**. The Concordium Ledger App only supports Ledger firmware version 2.1.0 on **Testnet**.
+Once you've set up the Ledger, you must check that it's running the proper firmware version. The Concordium Ledger App currently supports Ledger firmware version 2.1.0.
```

Comment:
For the Ledger Nano S we support 2.1.0, and for Ledger Nano S Plus we support 1.0.3.